src/Core/Addon/Module/ModuleManagerBuilder.php

```diff
@@ -159,13 +166,18 @@ private function getSymfonyRouter()
     {
         // get the environment to load the good routing file
         $routeFileName = _PS_MODE_DEV_ === true ? 'routing_dev.yml' : 'routing.yml';
-        $routesDirectory = _PS_ROOT_DIR_.DIRECTORY_SEPARATOR.'app'.DIRECTORY_SEPARATOR.'config';
+        $routesDirectory = $this->getConfigDir();
         $locator = new FileLocator(array($routesDirectory));
         $loader = new YamlFileLoader($locator);
 
         return new Router($loader, $routeFileName);
     }
 
+    protected function getConfigDir()
```

Comment:
why method for gettting config dir is in ModuleManager? Shouldn't this be somewhere higher? In some more generic class?